### PR TITLE
Misc changes

### PR DESCRIPTION
**Pull request by SpeedsterF2**
Updated to Quarkus 2.14.3 + some trivial changes

### CODE REVIEW

**Comment by bpasson** [comment]
Thanks for submitting a PR. I will review it and if all checks out accept it.

**Comment by SpeedsterF2** [comment]
rebased PR on current master

**Comment by bpasson** [comment]
@SpeedsterF2 can you have a look at the review comment?

**Comment by SpeedsterF2** [comment]
> @SpeedsterF2 can you have a look at the review comment?

Sorry, i do not konw where to look for comments added to the PR. Maybe that is just due to me being used to bitbucket.

**Comment by bpasson** [comment]
@SpeedsterF2 See the files tab when logged in to your account. You should see the comments then. https://github.com/quarkiverse/quarkus-rabbitmq-client/pull/138/files

**Comment by SpeedsterF2** [comment]
> @SpeedsterF2 See the files tab when logged in to your account. You should see the comments then. https://github.com/quarkiverse/quarkus-rabbitmq-client/pull/138/files

I do not see any. 
grafik

**Review by bpasson — CHANGES_REQUESTED** [review]

File: runtime/src/main/java/io/quarkiverse/rabbitmqclient/RabbitMQClients.java [hunk 1]
```diff
@@ -88,7 +92,11 @@ private RabbitMQClientParams params(String name) {
      */
     public void destroy() {
         clients.forEach((k, v) -> {
-            v.disconnect();
+            try {
```
[the remark this answers is not in the document]
Comment (bpasson) [reply runtime/src/main/java/io/quarkiverse/rabbitmqclient/RabbitMQClients.java:95]: @SpeedsterF2 do you see them now? The comments got stuck on pending it seemed.

[the remark this answers is not in the document]
Comment (SpeedsterF2) [reply runtime/src/main/java/io/quarkiverse/rabbitmqclient/RabbitMQClients.java:95]: Yeahr, now I see your comment!

**Comment by bpasson** [comment]
@SpeedsterF2 I am closing the PR as it has diverged from the master branch and I am cleaning up to move to Quarkus 3.

**Comment by SpeedsterF2** [comment]
OK,  I did not have time to work on this, lets reboot when v3 is out

**Pull request closed without merging**